docs/pre-release/reference/intercepts/index.md

```diff
@@ -68,14 +68,25 @@ your cluster; if it detects it correctly, may simply press "enter" and
 accept the default, otherwise you must tell Telepresence the correct
 value.
 
-When creating an intercept with the `http` mechanism, the
-traffic-agent sends a `GET /telepresence-http2-check` request to your
-service and to the process running on your local machine at the port
-specified in your intercept, in order to determine if they support
-HTTP/2.  This is required for the intercepts to behave correctly.  If
-you do not have a service running locally when the intercept is
-created, the traffic-agent will use the result it got from checking
-the in-cluster service.
+When creating an intercept with the `http` mechanism, Telepresence must
+determine the application protocol to use (HTTP/1.1 or HTTP/2). If the
+service's `ports.appProtocol` field is set, it will be used. If not, then
+Telepresence will use the configured [application protocol strategy](../config/#intercept)
+to determine the protocol. The default behavior (`http2Probe` strategy) is
+to send a `GET /telepresence-http2-check` request to your service in order
+to determine if it supports HTTP/2.  This is required for the intercepts to
+behave correctly.
```

Comment:
Just some minor tweaks:
_When you create an intercept with the http mechanism, Telepresence  determines whether the application protocol uses HTTP/1.1 or HTTP/2. If the service's ports.appProtocol field is set, Telepresence uses that. If not, then Telepresence uses the configured application protocol strategy to determine the protocol. The default behavior (http2Probe strategy) sends a GET /telepresence-http2-check request to your service to determine if it supports HTTP/2. This is required for the intercepts to behave correctly._